### PR TITLE
LRQA-80315 Define separate User Management test suite

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
[LRQA-80315](https://issues.liferay.com/browse/LRQA-80315)

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
ci:stop:security

**Comment by gaborlovasdotcom** [comment]
ci:test:um

**Comment by gaborlovasdotcom** [comment]
@melodywu, could you please review this PR?

File: test.properties [hunk 1]
```diff
@@ -8567,6 +8567,49 @@
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][translation]=\
         (testray.main.component.name ~ "Translations Management")
 
+    #
+    # UM
+    #
+
+    test.batch.class.names.includes[um]=\
+        **/account/**/*Test.java,\
+        **/address-test/**/*Test.java,\
```
Comment (JoyceWang08211) [inline test.properties:8576]: My only concern is that I'm unsure if the **address** component belongs to our team.

File: test.properties [hunk 2]
```diff
@@ -8567,6 +8567,49 @@
...
+        **/company-test/**/*Test.java,\
+        **/configuration-admin/**/*Test.java,\
+        **/expando-test/**/*Test.java,\
```
Comment (patriciajeanperez) [inline test.properties:8579]: @gaborlovasdotcom I believe these belong to Core/PEDS

**Review by patriciajeanperez — COMMENTED** [review]
Hi Gabor! LGTM. I would just start a conversation with your team to confirm if Address and On Demand Admin will fall under AppSec.

File: test.properties (hunk 2 above)
Comment (gaborlovasdotcom) [reply test.properties:8579]: Can you confirm @vicnate5?

**Comment by gaborlovasdotcom** [comment]
ci:test:um

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:stable

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:um

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Comment by gaborlovasdotcom** [comment]
ci:forward:force

**Pull request closed without merging**